docs/src/pages/discover-more/related-projects/related-projects.md

```diff
@@ -76,6 +76,10 @@ They support many different use cases (editable, filtering, grouping, sorting, s
 
 - [material-ui-confirm](https://github.com/jonatanklosko/material-ui-confirm): Provides easy to use confirmation dialogs to simplify confirming user actions without writing boilerplate code.
 
+### Molecules
```

Comment:
What do you think of this alternative name? We have started to study a couple of similar projects with different tech stack in https://trello.com/c/RdulDeBI/2195-building-blocks-organises. I love the initiative ❤️. 

```suggestion
### Blocks
```

Reply:
I have never realized that mdbootstrap had copied all the content of https://github.com/froala/design-blocks to rank #2 and make you think you need to pay for their material design themed version of bootstrap while it would also work without. It's borderline. I wonder how they get away with Google search's deduplication algorithm.

Anyway, I think that it's a good signal, there is something to explore here :)

Reply:
I even wonder if it's legal: mdbootstrap seems to break the license: https://github.com/froala/design-blocks/blob/dev/LICENSE.

Reply:
Totally fine for me, will make the change :) Thanks!